inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -549,6 +554,25 @@ TEST_P(IEClassNetworkTestP, QueryNetworkActualNoThrow) {
     }
 }
 
+TEST_P(IEClassNetworkTestP, QueryNetworkWithKSO) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    Core ie;
+
+    try {
+        auto rres = ie.QueryNetwork(ksoNetwork, deviceName);
+        auto rl_map = rres.supportedLayersMap;
+        auto func = ksoNetwork.getFunction();
+        for (const auto & op : func->get_ops()) {
+            if (!rl_map.count(op->get_friendly_name())) {
+                FAIL() << "Op " << op->get_friendly_name() << " is not supported by " << deviceName;
+            }
+        }
```

Comment:
could you please try to add `LoadNetwork` case on the network where affinities are used after `QueryNetwork` call

Reply:
Fixed